tests/unit/Tools/TaskRateCalculatorTest.php

```diff
@@ -23,8 +23,8 @@ public function testEmptyTasks()
     public function testTaskIncomingRate()
     {
         // make a task
-        $this->createTaskWithCreatedDate(1, '1970-01-01 01:30:00');
-        $this->createTaskWithCreatedDate(2, '1970-01-01 01:45:00');
+        $this->createTaskWithProcessedDate(1, '1970-01-01 01:30:00');
```

Comment:
this test should not change incomming counting was correct

Reply:
Reverted to created date.